Header header
float64 x
float64 y
float64 z
uint8 EditMode
uint8 DisplayIndices
uint8 DisplayConnections